docs/_pages/docs/03-understand/04-bonds.md

```diff
@@ -2,10 +2,54 @@
 layout: single
 title: Maker and Taker Bonds
 permalink: /docs/bonds/
+toc: true
+toc_sticky: true
 sidebar:
   title: '<img id="side-icon-verybig" src="/assets/vector/ticket-simple.svg"/>Bonds'
   nav: docs
 src: "_pages/docs/03-understand/04-bonds.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+The fidelity bond is a small deposit that the user "locks" which will be relinquished after the trade is completed; however, users can lose their bond if they fail to follow the obligations of the contract.
+
+The RoboSats trade pipeline utilizes fidelity bonds to incentivize both the order maker and taker to play by the rules and not cheat their fellow robot. More specifically, the bonds are hold invoices using the Lightning Network.
```

Comment:
I very much like these explanations! Hold invoices are cool and is the tech that makes RoboSats possible. Maybe we can add a link to some in depth technical document of how hodl invoices work?

Reply:
Yes definitely agree. Did you have a recommended link? I have stumbled across the two sources below (it seems I'm not yielding many search results for Lightning hold invoice docs):

https://docs.lightning.engineering/the-lightning-network/payment-lifecycle
https://lightningwiki.net/index.php/HODL_Invoice

Reply:
Those are good resources! Any work to me, I also like this one https://github.com/lightningnetwork/lnd/pull/2022